flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamContextEnvironment.java

```diff
@@ -38,14 +38,10 @@
 	private final ContextEnvironment ctx;
 
 	protected StreamContextEnvironment(ContextEnvironment ctx) {
+		super(GlobalConfiguration.loadConfiguration().getInteger(
```

Comment:
Or we could somehow hand the `Configuration` that the `CliFrontend` loads through to the `StreamingExecutionConfig`. I think that's a bigger, unrelated changed, however.